apps/product/src/main/java/com/ecommerce/product/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,28 @@
+package com.ecommerce.product.mapper;
+
+import com.ecommerce.product.dto.CategoryDTO;
+import com.ecommerce.product.model.Category;
+
+public class CategoryMapper {
+    public static CategoryDTO toDTO(Category category){
+        if (category == null) {
+            return null;
+        }
+
+        CategoryDTO dto = new CategoryDTO();
+        dto.setId(category.getId());
+        dto.setName(category.getName());
+        return dto;
+    }
+
+    public static Category toEntity(CategoryDTO dto) {
+        if (dto == null) {
+            return null;
+        }
+
+        Category category = new Category();
```

Comment:
```suggestion
        Category category = new Category().builder().setId(dto.getId()).setName(dto.getName()).build();
```

Using builder is better in this case.